src/core/a-entity.js

```diff
@@ -715,30 +716,6 @@ var proto = Object.create(ANode.prototype, {
       }
 
       /**
-       * Just update multiple component properties at once for a multi-property component.
```

Comment:
There was a lot of redundancy in that logic. The flag `clobber` basically replaces all of it